core/src/main/java/com/predic8/membrane/core/interceptor/HTTPClientInterceptor.java

```diff
@@ -73,10 +66,14 @@ public Outcome handleRequest(Exchange exc) throws Exception {
 			createAndSetErrorResponse(exc,502,msg);
 			return ABORT;
 		} catch (UnknownHostException e) {
-			String msg = format("Target host %s is unknown. DNS was unable to resolve host name.", URLUtil.getHost(getDestination(exc)));
-			log.warn(msg);
-			log.warn("Maybe the target is only reachable over an HTTP proxy server. Please check proxy settings in conf/proxies.xml.");
-			createAndSetErrorResponse(exc,500, msg);
+			String msg = "Target host %s of API %s is unknown. DNS was unable to resolve host name.".formatted(URLUtil.getHost(getDestination(exc)), exc.getRule().getName());
+			log.error(msg);
+			log.error("Maybe the target is only reachable over an HTTP proxy server. Please check proxy settings in conf/proxies.xml.");
```

Comment:
Multiple consecutive `log.error(...)` invocations are a bad pattern, as they are difficult to track with Log Aggregation utilities (Elasticsearch or Loki).
-> please concatenate the message, only use one `log.error(...)`